packages/terra-worklist-data-grid/CHANGELOG.md

```diff
@@ -1,9 +1,14 @@
 # Changelog
 
 ## Unreleased
+* Breaking Change
+  * Renamed `columnWidth` to `defaultColumnWidth`.
+  
 Added
  * Added additional keyboard navigation in `terra-worklist-data-grid`.
  * Added support for cell selection.
+ * Added ability to resize columns via mouse.
+ * Added `onColumnResize` prop to notify consumer of a column resize
```

Comment:
"Notify" seems inaccurate based on the prop description. Would something like the following be more accurate?
```suggestion
 * Added `onColumnResize` prop for consumers to be able to provide a callback function for when a column resize event occurs.
```

Reply:
It is not a resize event.  It is a callback that we create to notify the consumer that the column width was changed.